app/Theme/Scripts.php

```diff
@@ -59,4 +61,26 @@ public function addScriptTypeToJs($attr)
 
 		return $attr;
 	}
+
+	public function enqueueBlocksVariations()
+	{
+		wp_enqueue_script('blocks-variations', get_theme_file_uri('/assets/js/block-variations.js'), [
+			'wp-blocks',
+			'wp-dom',
+			'wp-edit-post',
+		], '', true);
+	}
+
+	public function enqueueBlockStyleVariations()
+	{
+		wp_enqueue_script(
+			'block-style-variations',
+			get_theme_file_uri('/assets/js/block-style-variations.js'),
+			[
+				'wp-blocks',
+				'wp-dom-ready',
+				'wp-edit-post'
+			]
+		);
+	}
```

Comment:
**nitpick:** let's make the formatting and arguments consistent (unless the difference is intentional?), between these two.

Reply:
I've tidied up the formatting. I've swapped one of the dependencies for the variations script to call a more limited one, but the rest is as suggested in the documentation.